README.md

```diff
@@ -435,6 +435,21 @@ go mod tidy
 go mod vendor
 ```
 
+## Building without network acess
+
+Go modules require network acess. If you need to make a repetible build with
+no network acess, make sure that your code is under `$GOPATH` and run the
+following:
+
+``shell
+mkdir -p /tmp/gopath/src/github.com/u-root/
```

Comment:
People are going to assume you HAVE to do lines 445-448, but there are several ways to get u-root under GOPATH, for example:
GOPATH=~/go GO111MODULE=off go get github.com/u-root/u-root
works fine. You might want to mention lines 445-448 under "for example", but even then, if you do this
GOPATH=/tmp/go GO111MODULE=off go get github.com/u-root/u-root
you can save a lot of typing.
And note, the GO111MODULE=off AND GOPATH are both needed.